pss/pss_test.go

```diff
@@ -507,116 +510,128 @@ func TestMessageProcessing(t *testing.T) {
 
 	addr := make([]byte, 32)
 	addr[0] = 0x01
-	ps := newTestPss(privkey, network.NewKademlia(addr, network.NewKadParams()), NewParams())
+	ps := newTestPssStart(privkey, network.NewKademlia(addr, network.NewKadParams()), NewParams(), false)
 	defer ps.Stop()
-
-	// message should pass
-	msg := newPssMsg(&msgParams{})
-	msg.To = addr
-	msg.Expire = uint32(time.Now().Add(time.Second * 60).Unix())
-	msg.Payload = &whisper.Envelope{
-		Topic: [4]byte{},
-		Data:  []byte{0x66, 0x6f, 0x6f},
+	outboxCapacity := 2
+
+	processed := make([]*PssMsg, 0)
```

Comment:
no need for this just time out on successC instead

Reply:
FIxed in 1b91b66